include/CppUTest/CppUTestConfig.h

```diff
@@ -216,8 +218,6 @@ typedef struct { char dummy[16]; } cpputest_ulonglong;
 typedef struct { char dummy[8]; } cpputest_longlong;
 typedef struct { char dummy[8]; } cpputest_ulonglong;
 #endif
-#define CPPUTEST_LONGLONG_DEFAULT cpputest_longlong()
-#define CPPUTEST_ULONGLONG_DEFAULT cpputest_ulonglong()
```

Comment:
The coverage decrease is because you put the `#ifdef`s back around the macros, and removed the corresponding macro tests, so when `CPPUTEST_USE_LONG_LONG` is not defined it's not exercising `UtestShell::assertLongLongsEqual` any more.  I had extra unit tests in UtestShellTest for that case; want me to dig them up?

The other cause is that coverage is being done without long-long support, because that uses the CMake build and the CMake build hasn't yet had config.h renamed to generated/CppUTestGeneratedConfig.h, so CMake's options aren't included any more.